packages/shared/lib/core/error/constants/default-user-error-parameters.constant.ts

```diff
@@ -0,0 +1,8 @@
+import { IErrorParameters } from '../interfaces'
+
+export const DEFAULT_USER_ERROR_PARAMETERS: Partial<IErrorParameters> = {
+    localizeMessage: true,
+    showNotification: true,
+    saveToErrorLog: true,
```

Comment:
default user error shouldn't be to save to error log?

Reply:
Yeah, that makes sense. This should probably be called "app error" and not user